ldcoolp/scripts/generate_wccfl_qualtrics_links

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+from os.path import dirname, exists, join
+from os import mkdir, stat
+
+from urllib.parse import quote, urlencode
+import json
+import requests
+
+import csv
+
+import pandas as pd
+
+import argparse
+
+from datetime import date
+
+from ldcoolp.logger import LogClass, get_user_hostname
+from ldcoolp.curation.api import figshare
+from ldcoolp.curation import df_to_dict_single
+from ldcoolp.admin import permissions
+from ldcoolp.config import dict_load
+
+from ldcoolp.curation.email import urls
+
+# Version and branch info
+from ldcoolp import __version__
+from ldcoolp.git_info import get_active_branch_name, get_latest_commit
+from ldcoolp import __file__ as library_path
+
+today = date.today()
+
+library_root_path = dirname(dirname(library_path))  # Retrieve parent directory to ldcoolp
+
+url_safe = '/ {},:"?=@%'
+
+# Custom settings for script
+wccfl_survey_id = 'SV_9KOrfOnPuXgVIDc'
+account_id = 2641202
+
+
+def get_articles_list(fs_dict, article_id=''):
+    """Get list of private articles under depositor account for WCCFC
+    """
+    if not article_id:
+        url = 'https://api.figshare.com/v2/account/articles'
+    else:
+        url = f'https://api.figshare.com/v2/account/articles/{article_id}'
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Authorization': f'token {fs_dict["api_token"]}'
+    }
+
+    params = {'impersonate': account_id}
+    if not article_id:
+        params['page'] = 1
+        params['page_size'] = 100
+
+    response = requests.get(url, headers=headers, params=params)
+
+    if not article_id:
+        return pd.DataFrame(response.json())
+    else:
+        return response.json()
+
+
+def wccfl_generate_url(qualtrics_dict, dn_dict):
+    populate_response_dict = dict()
+    for query in ['QID4', 'QID6']:
+        populate_response_dict[query] = {
+            "1": dn_dict['full name'],
+            "2": dn_dict['email'],
+            "3": dn_dict['affiliation']
+        }
+
+    populate_response_dict['QID7'] = dn_dict['title']
+
+    json_txt = quote(json.dumps(populate_response_dict), safe=url_safe)
+
+    query_str_dict = {'article_id': dn_dict['article_id'],
+                      # 'curation_id': dn_dict['curation_id'],
+                      'Q_PopulateResponse': json_txt}
+
+    full_url = f"{qualtrics_dict['generate_url']}{wccfl_survey_id}?" + \
+               urlencode(query_str_dict, safe=url_safe, quote_via=quote)
+    return full_url
+
+
+if __name__ == '__main__':
+    # Parse command-line arguments
+    parser = argparse.ArgumentParser(
+        description='Command-line driver for LD-Cool-P retrieval of WCCFL Qualtrics links.')
+    parser.add_argument('--config', required=True,
+                        help='path to configuration file')
+    parser.add_argument('--csv-file', required=True,
+                        help='CSV file containing list of presenters')
+    parser.add_argument('--tiny_url', action='store_true',
```

Comment:
My commit 831a040. There was a typo: `'--tinyurl'`

Reply:
`--tiny-url` instead since the convention it to use "tag" with parsing arguments